### PR TITLE
cloud: add release notes 20230808

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
 
+## August 8, 2023
+
+**General changes**
+
+- [Data Service](https://tidbcloud.com/console/data-service) now supports Basic Authentication.
+
+    You can provide your public key as the username and private key as the password in requests using standard [HTTP Basic Authentication](https://datatracker.ietf.org/doc/html/rfc7617). Compared with Digest Authentication, Basic Authentication is simpler, enabling more straightforward usage when calling Data Service endpoints.
+
+     For more information, see [Call an endpoint](/tidb-cloud/data-service-manage-endpoint.md#call-an-endpoint)
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:19]:
```suggestion
     For more information, see [Call an endpoint](/tidb-cloud/data-service-manage-endpoint.md#call-an-endpoint).
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,16 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
 
+## August 8, 2023
+
+**General changes**
+
+- [Data Service](https://tidbcloud.com/console/data-service) now supports Basic Authentication.
+
+    You can provide your public key as the username and private key as the password in requests using standard [HTTP Basic Authentication](https://datatracker.ietf.org/doc/html/rfc7617). Compared with Digest Authentication, Basic Authentication is simpler, enabling more straightforward usage when calling Data Service endpoints.
+
+     For more information, see [Call an endpoint](/tidb-cloud/data-service-manage-endpoint.md#call-an-endpoint).
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:19]:
```suggestion
    For more information, see [Call an endpoint](/tidb-cloud/data-service-manage-endpoint.md#call-an-endpoint).
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 3]
```diff
@@ -8,6 +8,16 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## August 8, 2023
+
+**General changes**
+
+- [Data Service](https://tidbcloud.com/console/data-service) now supports Basic Authentication.
+
+    You can provide your public key as the username and private key as the password in requests using standard [HTTP Basic Authentication](https://datatracker.ietf.org/doc/html/rfc7617). Compared with Digest Authentication, Basic Authentication is simpler, enabling more straightforward usage when calling Data Service endpoints.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:17]:
```suggestion
    You can provide your public key as the username and private key as the password in requests using the [Basic HTTP Authentication](https://datatracker.ietf.org/doc/html/rfc7617). Compared with Digest Authentication, Basic Authentication is simpler, enabling more straightforward usage when calling Data Service endpoints.
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**